### DISCUSSION TITLE
Correct the `Used by` number in repo homepage

### DISCUSSION THREAD

**Discussion by chenjiahan** [swc-project/swc#9115] [General]
Currently the `Used by` not truly reflect the number of projects using SWC:

![image](https://github.com/swc-project/swc/assets/7237365/d976e3de-d4b3-4bee-874c-19be6f8f6d33)

I recommend showing the dependent count of `@swc/core`, which should be `464k` instead of `6k`.

![image](https://github.com/swc-project/swc/assets/7237365/88c002cc-53f8-4211-b6af-c5a53554af8a)

You can go to the `Code security and analysis` setting page to configure it:

![image](https://github.com/swc-project/swc/assets/7237365/4b91dd13-d468-40fc-bbb4-f7b5d88d51f0)

@kdy1 cc

**Comment by kdy1** [comment]
I configured it. Thank you! I didn't know that I can change it.
